tests/e2e-tests/scripts/run-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Could we maybe add one or two lines documenting what is this script purpose? I think it would be useful to do the same to `sp-config.sh`, `wp-serverpilot-delete.js` and `wp-serverpilot-init.js`

Also, I'm unsure if this script should be in this directory with the rest of the e2e related scripts or in `tests/bin` where we keep all the other scripts that are called by Travis. Thoughts?